openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -452,15 +452,45 @@ <h3 translate>Version history</h3>
 </div>
 
 <div class="details" ng-if="isAllowedToSeeCommentField()">
-  <h3 translate>Comments</h3>
-  <div ng-repeat="field in commentsFields">
-    <div ng-if="(field.public && !field.forState && !field.forRecommendation) ||
-        (operator.hasPerms('motions.can_see_and_manage_comments') && !field.forState && !field.forRecommendation)">
-      <h4>
-        {{ field.name }}
-        <span ng-if="!field.public" class="label label-warning" translate>internal</span>
-      </h4>
-      <div ng-bind-html="motion.comments[$index]"></div>
+  <div class="row">
```

Comment:
Can you put this stuff into an extra file and use ng-include? Like we do for normal text editing and toolbar.